recipes/jove/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "jove" %}
+{% set version = "4.17.3.6" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/jonmacs/jove/releases/download/{{ version }}/jove-{{ version }}.tgz
+  sha256: 94300bed2b73b3b964fe9e42c652175525609d3ebf6fdefd2291d277b5ae27a2
+
+build:
+  number: 0
+  skip: true  # [win]
+  script: 'make -f Makefile.msc && cp jjove.exe "$PREFIX/bin/jove.exe"'  # [win]
+  script: 'make JOVEHOME="$PREFIX" SYSDEFS=-DDarwin install'  # [osx]
+  script: 'make JOVEHOME="$PREFIX" LDFLAGS="$LDFLAGS " TERMCAPLIB=-lncurses install'  # [linux]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - ncurses  # [not win]
+
+test:
+  commands:
+    - which jove
+
+about:
+  home: https://github.com/jonmacs/jove/
+  license: LicenseRef-JOVE
+  license_file: README
```

Comment:
Maybe you could extract the license into a separate file in the recipe and use that?

Reply:
You think the text from the start of the README as a file in the recipe repository would suffice?

```
##########################################################################
# This program is Copyright (C) 1986-2002 by Jonathan Payne.  JOVE is    #
# provided by Jonathan and Jovehacks without charge and without          #
# warranty.  You may copy, modify, and/or distribute JOVE, provided that #
# this notice is included in all the source files and documentation.     #
##########################################################################
```

(Perhaps without the ``#`` signs)